_drafts/semantic-release.md

```diff
@@ -0,0 +1,37 @@
+---
+state: In Progress
+---
+
+# Set up release workflow with `semantic-release`
+
+`semantic-release` makes releasing faster, simpler and safer. All of these are valuable if we want to increase public participation in the project and increase momentum and motivation.
+
+## Motivation
+
+Releasing a package is always a bit of a Big Thing™: decisions have to be made about versioning, an error-prone manual release process has to be performed, and somebody has to actually do the work and take the responsibility if something goes wrong. So there are a few barriers to frequent releases, and  many potential sources of error. Automating this process takes away all the uncertainty and hassle out of releasing, makes our version numbers more consistent and reliable, and increases release frequency and project momentum. It also does away with the concept of intentional, manual releases: whenever something lands in master and passes all the tests, `semantic-release` will package an npm release. If we do need to keep control of release timing for marketing reasons, `semantic-release` supports that too.
```

Comment:
> It also does away with the concept of intentional, manual releases: whenever something lands in master and passes all the tests, `semantic-release` will package an npm release

This is really cool. This is pretty close to how I worked previously when making changes, basically timeboxing a day to see how much I could merge and then release that with whatever semantically appropriate version.